docs/source/installation/pc-testing-setup.rst

```diff
@@ -7,45 +7,44 @@ PC Testing
 JDK Installation
 -----------------
 
-The program requires JDK Version 12 or newer.
+Chameleon Vision requires JDK Version 12 or newer.
+To check whether you are running a compatible version, type ``$ java --version``. 
+You should see a result similar to the one below:
 
-How do you know that version are you running?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: console
 
-Run ``java -version`` in your command line. You should see a similar result:
+	$ java --version
+	java version "12.0.2" 2019-07-07
+	Java(TM) SE Runtime Environment (build 12.0.2+10)
+	Java HotSpot(TM) 64-bit Server VM (build 12.0.2+10, mixed mode, sharing)
 
-.. image:: /images/installation/java-version.png
+If your version is older than JDK 12, follow the instructions below.
 
 Installing JDK 12
 ^^^^^^^^^^^^^^^^^^^
 
 Windows
-~~~~~~~~
-
-Download JDK 12 from this `link <https://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ and install .
+~~~~~~~~~
 
+Download and run the installer from the `oracle website <https://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ and follow the installation instructions.
```

Comment:
Suggest OpenJDK/Bellsoft instead of Oracle. Oracle now requires users to sign up to get downloads.